app/presenters/result_set_presenter.rb

```diff
@@ -64,4 +65,8 @@ def documents
       SearchResultPresenter.new(result).to_hash
     end
   end
+
+  def atom_url
+    params.empty? ? "#{finder.slug}.atom" : "#{finder.slug}.atom?#{params.to_query}"
```

Comment:
I'm now much more confident that there won't be unintended side effects from this :+1: